src/collection-view.js

```diff
@@ -549,14 +561,26 @@ const CollectionView = Backbone.View.extend({
   // This will also update the indices of later views in the collection in order to keep the
   // children in sync with the collection.
   addChildView(view, index) {
+    if (!(view instanceof Backbone.View)) {
```

Comment:
actually, after reading https://github.com/marionettejs/backbone.marionette/pull/3155#issuecomment-245181213 I'll remove.
If it becomes an issue and the check is need, easy to update.